pyproject.toml

```diff
@@ -1,3 +1,113 @@
+description = "A standard API for reinforcement learning and a diverse set of reference environments (formerly Gym)."
+readme = "README.md"
+requires-python = ">= 3.7"
+authors = [{ name = "Farama Foundation", email = "contact@farama.org" }]
+license = { text = "MIT License" }
+keywords = ["Reinforcement Learning", "game", "RL", "AI", "gymnasium"]
+classifiers = [
+    "Development Status :: 4 - Beta",  # change to `5 - Production/Stable` when ready
+    "License :: OSI Approved :: MIT License",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    'Intended Audience :: Science/Research',
+    'Topic :: Scientific/Engineering :: Artificial Intelligence',
+]
+dependencies = [
+    "numpy >=1.21.0",
+    "jax-jumpy >=0.2.0",
+    "cloudpickle >=1.2.0",
+    "importlib-metadata >=4.8.0; python_version < '3.10'",
+    "typing-extensions >=4.3.0; python_version == '3.7'",
+    "gymnasium-notices >=0.0.1",
+    "shimmy >=0.1.0,<1.0",
+]
+dynamic = ["version"]
+
+[project.optional-dependencies]
+# Update dependencies in `all` if any are added or removed
+atari = ["shimmy[atari] >=0.1.0,<1.0"]
+accept-rom-license = ["autorom[accept-rom-license] ~=0.4.2"]
+box2d = ["box2d-py ==2.3.5", "pygame ==2.1.0", "swig ==4.*"]
+classic-control = ["pygame ==2.1.0"]
+classic_control = ["pygame ==2.1.0"]  # kept for backward compatibility
+mujoco-py = ["mujoco-py >=2.1,<2.2"]
+mujoco_py = ["mujoco-py >=2.1,<2.2"]  # kept for backward compatibility
+mujoco = ["mujoco >=2.3.0", "imageio >=2.14.1"]
+toy-text = ["pygame ==2.1.0"]
+toy_text = ["pygame ==2.1.0"]         # kept for backward compatibility
+jax = ["jax ==0.3.20", "jaxlib ==0.3.20"]
+other = [
+    "lz4 >=3.1.0",
+    "opencv-python >=3.0",
+    "matplotlib >=3.0",
+    "moviepy >=1.0.0",
+    "tensorflow >=2.1.0",
+    "torch >=1.0.0",
+]
+all = [
+    # All dependencies above except accept-rom-license
+    # NOTE: No need to manually remove the duplicates, setuptools automatically does that.
+    # atari
+    "shimmy[atari] >=0.1.0,<1.0",
+    # box2d
+    "box2d-py ==2.3.5",
+    "pygame ==2.1.0",
+    "swig ==4.*",
+    # classic-control
+    "pygame ==2.1.0",
+    # mujoco-py
+    "mujoco-py >=2.1,<2.2",
+    # mujoco
+    "mujoco >=2.3.0",
+    "imageio >=2.14.1",
+    # toy-text
+    "pygame ==2.1.0",
+    # jax
+    "jax ==0.3.20",
+    "jaxlib ==0.3.20",
+    # other
+    "lz4 >=3.1.0",
+    "opencv-python >=3.0",
+    "matplotlib >=3.0",
+    "moviepy >=1.0.0",
+    "tensorflow >=2.1.0",
+    "torch >=1.0.0",
+]
+testing = ["pytest ==7.1.3"]
+
+[project.urls]
+Homepage = "https://farama.org"
+Repository = "https://github.com/Farama-Foundation/Gymnasium"
+Documentation = "https://gymnasium.farama.org"
+"Bug Report" = "https://github.com/Farama-Foundation/Gymnasium/issues"
```

Comment:
Should the `Homepage` be `"https://gymnasium.farama.org"`, the same as `Documentation`?